### ISSUE TITLE
[High Sierra] Reboot when loading installer

### ISSUE THREAD

**Issue by elosha** [d4vinder/HP-Z620-Hackintosh-macOS-Catalina#10]
Hey and thanks for your work!

I want to put High Sierra (10.13) on my Z620. I have a E5-2620 v0 CPU and a K2000 GPU (should work natively). Followed the guide exactly, except for making a HS Installer instead.

a) @d4vinder requested "SAS Controller = Hidden", I don't have this setting, but "SCU = Hidden" (SAS Controller Unit). I also don't have the Serial/Diskette settings … I believe this was from Z600? Can this be a showstopper if missed?

b) I guess High Sierra should work, too?

c) Which CPU architecture is your guide about? I read that v2 CPUs can appear in some Z620 and require different power management settings than v0.

d) I currently only use a old 1280x1024 VGA screen, so I changed the config from 1920x1280. Nevertheless, most of the times I don't see the Clover screen to change boot params – why?

e) When the installer attempts to boot, I see the Apple logo, but instantly after the loading bar appears, the machine reboots. I read that exchanging your VBoxHfs.efi driver for the HfsPlus.efi driver resolves that issue, but it didn't for me :(

f) Is it crucial to use Clover 5120? I used 5126 (latest atm)

Hope someone can help!

**Comment by d4vinder** [comment]
Use 5120. 5126 doesn't work for some reason.

**Comment by elosha** [comment]
Sadly, still the same result. :(

I'm using a E5-2650-v2 now, Clover r5120 and a 1920x1280 display. Still using High Sierra (did you get it to work?) and K2000.

I tried verbose mode now. Kernel loads very quickly, then various things happen:

First, I needed to use [a different USBInjectAll](https://github.com/Sniki/OS-X-USB-Inject-All/issues/7) due to a linking error, which could be resolved.

Then in between I get
`ACPI Error: [_SB_.PCI0.GFX0] namespace lookup failure`

After "ACPI: Sleep status S3 S4 S5" the kernel panics with a page fault 
It lists
```
Kernel Extensions in backtrace:
[…] IOACPIFamiliy […]
[…] AppleACPIPlatform(6.1) […]
```
and
`Last loaded kext at […]: >!ASMBIOS 2.1`

No non-stock hardware.
